packages/bitcore-wallet-service/src/lib/model/txproposal.ts

```diff
@@ -410,18 +414,23 @@ export class TxProposal {
     this._updateStatus();
   }
 
-  sign(copayerId, signatures, xpub) {
+  sign(copayerId, signatures, xpub, bchSchnorr) {
```

Comment:
bchSchnorr is a flag, so it should be named:

useBchSchnoor or bchScnoorSupported.

Maybe is better to pass `opts`
and then  `opts.useBchSchnoor`